src/main/java/alien4cloud/paas/yorc/context/service/fsm/FsmMapper.java

```diff
@@ -83,7 +88,12 @@ public static FsmStates fromYorcToFsmState(String status) throws Exception {
             case "DEPLOYMENT_IN_PROGRESS":
                 return FsmStates.DEPLOYMENT_IN_PROGRESS;
             case "DEPLOYMENT_FAILED":
-                return FsmStates.FAILED;
```

Comment:
Why is the "DePLOYMENT_FAILED" state now mapped to UPDATE_IN_PROGRESS FSM State?

Reply:
I think its an oversite... 
I can fix it if you think its necessary. Stephane its in vacation until early september, so we cannot wait for him to discuss.

Reply:
Yes i think it needs a fix

Reply:
Fixed.